jolt-core/src/jolt/vm/ram.rs

```diff
@@ -630,44 +636,55 @@ impl<F: JoltField, ProofTranscript: Transcript> ReadWriteCheckingProof<F, ProofT
         let span = tracing::span!(tracing::Level::INFO, "compute checkpoints");
         let _guard = span.enter();
 
-        // #[cfg(test)]
-        // let mut val_test: MultilinearPolynomial<F> = {
-        //     // Compute Val in cycle-major order, since we will be binding
-        //     // from low-to-high starting with the cycle variables
-        //     let mut val: Vec<u64> = vec![0; K * T];
-        //     val.par_chunks_mut(T).enumerate().for_each(|(k, val_k)| {
-        //         let mut current_val = initial_memory_state[k] as u64;
-        //         for j in 0..T {
-        //             val_k[j] = current_val;
-        //             if let RAMAccess::Write(write) = trace[j].ram_access() {
-        //                 if remap_address(write.address, memory_layout) == k as u64 {
-        //                     current_val = write.post_value;
-        //                 }
-        //             }
-        //         }
-        //     });
-        //     MultilinearPolynomial::from(val)
-        // };
-        // #[cfg(test)]
-        // let mut ra_test = {
-        //     // Compute ra in cycle-major order, since we will be binding
-        //     // from low-to-high starting with the cycle variables
-        //     let mut ra: Vec<F> = unsafe_allocate_zero_vec(K * T);
-        //     ra.par_chunks_mut(T).enumerate().for_each(|(k, ra_k)| {
-        //         for j in 0..T {
-        //             if remap_address(trace[j].ram_access().address() as u64, memory_layout)
-        //                 == k as u64
-        //             {
-        //                 ra_k[j] = F::one();
-        //             }
-        //         }
-        //     });
-        //     MultilinearPolynomial::from(ra)
-        // };
+        #[cfg(test)]
```

Comment:
I left these commented out because for the end-to-end tests, computing these polynomials is quite slow/memory-intensive